contracts/src/token/erc20/extensions/metadata.rs

```diff
@@ -56,9 +56,6 @@ pub trait IErc20Metadata {
     fn decimals(&self) -> u8;
 }
 
-// FIXME: Apply multi-level inheritance to export Metadata's functions.
-// With the current version of SDK it is not possible.
-// See https://github.com/OffchainLabs/stylus-sdk-rs/pull/120
```

Comment:
We know multi-level inheritance is not the direction the SDK is going, and we found away around this that I think is acceptable, so I'm ok removing this without creating an issue

Reply:
IMHO yes.